### PR TITLE
add little impl

### PR DESCRIPTION
**Pull request by karsv**

### CODE REVIEW

File: src/main/java/com/task/amazon/Main.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.task.amazon;
+
+import com.task.amazon.utils.ReaderService;
+import com.task.amazon.utils.impl.FileReaderService;
+
+import java.util.List;
+
+public class Main {
```
Comment (boroda4436) [inline src/main/java/com/task/amazon/Main.java]: @karsv isn't it a SpringBoot app? We should have only one main method. If this is a test case - please move it to a corresponding folder

File: src/main/java/com/task/amazon/entities/AmazonEntity.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "entities")
+public class AmazonEntity {
```
Comment (boroda4436) [inline src/main/java/com/task/amazon/entities/AmazonEntity.java]:
```suggestion
public class AmazonReviewEntity {
```

File: src/main/java/com/task/amazon/entities/AmazonEntity.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String productId;
```
Comment (boroda4436) [inline src/main/java/com/task/amazon/entities/AmazonEntity.java]: @karsv could we use this as Id? Or the 'productId' field is not unique?

File: src/main/java/com/task/amazon/utils/ParseService.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package com.task.amazon.utils;
+
+public interface ParseService {
```
Comment (boroda4436) [inline src/main/java/com/task/amazon/utils/ParseService.java]: @karsv do not push empty classes or interfaces in PR. Let's add them in the next PR as well as their implementations

File: src/main/java/com/task/amazon/utils/ReaderService.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package com.task.amazon.utils;
+
+import java.util.List;
+
+public interface ReaderService {
+    List<String> parse(String path);
```
Comment (boroda4436) [inline src/main/java/com/task/amazon/utils/ReaderService.java]: 'parse' method name is too general. can we rename this method name?

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/com/task/amazon/entities/AmazonEntity.java (hunk 3 above)
Comment (karsv) [reply src/main/java/com/task/amazon/entities/AmazonEntity.java]: We can use id's from csv file.

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**